test/foundry/BlastYield.t.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IOwnableTwoSteps} from "../../contracts/interfaces/IOwnableTwoSteps.sol";
+import {OwnableTwoSteps} from "../../contracts/OwnableTwoSteps.sol";
+import {BlastYield} from "../../contracts/BlastYield.sol";
+import {Test} from "../../lib/forge-std/src/Test.sol";
+import {YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "../../contracts/interfaces/IBlast.sol";
+import {YieldMode as IERC20Rebasing__YieldMode} from "../../contracts/interfaces/IERC20Rebasing.sol";
+
+import {MockERC20} from "../mock/MockBlastERC20.sol";
+import {MockPoints} from "../mock/MockBlastPoints.sol";
+import {MockWETH} from "../mock/MockBlastWETH.sol";
+import {MockYield} from "../mock/MockBlastYield.sol";
+
+contract BlastYieldOwnableTwoSteps is BlastYield, OwnableTwoSteps {
+    constructor(
+        address _blast,
+        address _blastPoints,
+        address _blastPointsOperator,
+        address _owner,
+        address _usdb,
+        address _weth
+    ) BlastYield(_blast, _blastPoints, _blastPointsOperator, _owner, _usdb, _weth) OwnableTwoSteps(_owner) {}
+
+    function claim(address wethReceiver, address usdbReceiver) public onlyOwner {
+        _claim(wethReceiver, usdbReceiver);
+    }
+}
+
+contract BlastYieldOwnableTwoSteps_Test is Test {
+    MockWETH private weth;
+    MockERC20 private usdb;
+    MockYield private mockYield;
+    MockPoints private mockPoints;
+    BlastYieldOwnableTwoSteps private blastYieldOwnableTwoSteps;
+
+    address public owner = address(69);
+    address public operator = address(420);
+    address public user1 = address(1);
+    address private constant TREASURY = address(69420);
+    address internal constant BLAST_POINTS = 0x2536FE9ab3F511540F2f9e2eC2A805005C3Dd800;
```

Comment:
We have a mock points contract right?

Reply:
we don't need this at all, just pass address(mockPoints) in L53

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/25174449f39c8926683a1fed441d3ca3f9ed5424